metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/HiveConnectorFastTableService.java

```diff
@@ -159,11 +170,36 @@ public void update(final ConnectorRequestContext requestContext, final TableInfo
             } else {
                 directSqlTable.updateIcebergTable(tableInfo);
             }
+        } else if (connectorContext.getConfig().isCommonViewEnabled()
```

Comment:
We should refactor this to handle common-view separately in a different method (handleCommonView). Also refactor the iceberg code to handleIcebergTable)

Reply:
Yes, refactored as suggested.